e2e/cypress/integration/menus/menu_actions_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```

Comment:
Please remove prod stage.

Reply:
> Thanks @openmohan! The main verification here is the box-shadow of the team icon. Not sure if there's a way to hold/mimic click event so there's enough time to verify the requirement.

I have tried, `.trigger('mousedown', {button: 0})`, it seems to be not changing the CSS aspects of the element. But the element is pressed with double click. Is this is a known scenario? if so, I can try other methods or API. Thanks!

Reply:
No, not known to me (not sure with others) until now after I read the test case. Yeah, looks like such approach didn't work.